rsdockerspawner/rsdockerspawner.py

```diff
@@ -591,9 +590,14 @@ def _no_slots(pool):
             return s, pool
 
     @classmethod
-    def __slot_assign(cls, slot, cname):
-        slot.activity_secs = time.time()
-        slot.start_time = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+    def __slot_assign(cls, slot, cname, previous_slot=None):
+        a = time.time()
```

Comment:
Put in an `else`. There's work being done here, and no reason to have it this way.

Reply:
0b2f009